patches/estdlib/supervisor.erl

```diff
@@ -60,3 +61,25 @@ init_children(State, StartSpec) ->
     Error ->
       {stop, {start_spec, Error}}
   end.
+
+% Patch reason: monotonic_time doesn't handle integer time units
+add_restart(State) ->
+  I = State#state.intensity,
+  P = State#state.period,
+  R = State#state.restarts,
+  % Replaced code
+  % Now = erlang:monotonic_time(1),
+  Now = erlang:monotonic_time(second),
```

Comment:
Can we patch monotonic_time itself?

Reply:
Nope, it's a NIF so the patch won't work. However, the change (in FissionVM) seems trivial, but I didn't want to delay this PR